srcalc/src/test/java/gov/va/med/srcalc/web/view/admin/EditVariableValidatorTest.java

```diff
@@ -39,6 +39,30 @@ public final void testValid()
 
         assertEquals("error count", 0, errors.getErrorCount());
     }
+    
+    @Test
+    public final void testKeyTooLong()
+    {
+        final EditVariable ev = makeEditVariable();
+        ev.setKey(TestHelpers.stringOfLength(Variable.KEY_MAX + 1));
+        final BindingResult errors = validate(ev);
+        
+        assertEquals(
```

Comment:
I would also test to ensure there are no other field errors besides the key length error.